### PR TITLE
Added a heading for legal considerations

### PR DESCRIPTION
**Pull request by GeorgeKerscher**
I added this just below the publisher metadata workflow  image.

### CODE REVIEW

File: UX-Guide-Metadata/draft/principles/index.html [hunk 1]
```diff
@@ -183,6 +183,16 @@ <h2>Publisher metadata ecosystem</h2>
                         <p>
                             Finally from the "Metadata Processing" layer both the Display Techniques for EPUB and ONIX flow into the "Resulting Statements" layer "User Experience Guide for Displaying Accessibility Metadata"</p>
                     </details>
+</section>
+
+<section id="legal-considerations">
+<h2>Legal Considerations</h2>
+
+<p> in some jurisdictions  publishers may have an exemption from the provision of accessible publications, including the provision of accessibility metadata. Some examples of this are when the publisher claims a disproportionate burden  exemption or when it is a micro-enterprise (e.g., enterprises employing fewer than 10 people and with annual
```
Comment (clapierre) [inline UX-Guide-Metadata/draft/principles/index.html:191]: Small typo, start of sentence "In" should be capitalized.

File: UX-Guide-Metadata/draft/principles/index.html [hunk 2]
```diff
@@ -183,6 +183,16 @@ <h2>Publisher metadata ecosystem</h2>
...
+turnover or balance sheet total not exceeding Γé¼2 million).</p>
```
Comment (clapierre) [inline UX-Guide-Metadata/draft/principles/index.html:192]: The exceeding amount in millions got mangled.

File: UX-Guide-Metadata/draft/principles/index.html (hunk 1 above)
Comment (mattgarrish) [reply UX-Guide-Metadata/draft/principles/index.html]: If you're citing the definition of a microenterprise, it should be "i.e.," not "e.g.,". Using "e.g." suggests there are other kinds of microenterprises beyond the kind listed.

File: UX-Guide-Metadata/draft/principles/index.html [hunk 3]
```diff
@@ -184,6 +184,14 @@ <h2>Publisher metadata ecosystem</h2>
                             Finally from the "Metadata Processing" layer both the Display Techniques for EPUB and ONIX flow into the "Resulting Statements" layer "User Experience Guide for Displaying Accessibility Metadata"</p>
                     </details>
 
+<h2>Legal Considerations</h2>
+
+<p> in some jurisdictions  publishers may have an exemption from the provision of accessible publications, including the provision of accessibility metadata. Some examples of this are when the publisher claims a disproportionate burden  exemption or when it is a micro-enterprise (e.g., enterprises employing fewer than 10 people and with annual
```
Comment (gautierchomel) [inline UX-Guide-Metadata/draft/principles/index.html]: This criteria for micro is for the EAA, may be different in other legislation. I would prefer not to include any example, eventually a note pointing to the EAA and later on other legislation.

File: UX-Guide-Metadata/draft/principles/index.html [hunk 4]
```diff
@@ -184,6 +184,14 @@ <h2>Publisher metadata ecosystem</h2>
...
+turnover or balance sheet total not exceeding Γé¼2 million).</p>
+
+<p >In cases    where the publisher is claiming an exemption it is up to  implementors to decide what to present to end users. Publishers may not want their exemption information to be made public. It may be advisable for implementors to instead provide guidance to the end user by including a user-friendly statement such as, "This title may not yet be fully accessible to all readers. Check the list of accessibility features to determine  if this title is suitable for your way of reading."</p>
```
Comment (gautierchomel) [inline UX-Guide-Metadata/draft/principles/index.html:194]: I would delete "it is up to  implementers to decide what to present to end users." to keep the sentence more clear on the fact that "Publishers may not want". 
New proposed sentence: "In cases where the publisher is claiming an exemption, it does not mean that they want their exemption information to be made public.
(again, if I risk that one implementer displays that, then I will not use the metadata)

**Review by gautierchomel — CHANGES_REQUESTED** [review]

**Comment by GeorgeKerscher** [comment]
Now I have a conflict, darn. I will wait until we have discussed.

**Pull request closed without merging**